### PR TITLE
Add friends-v1 feed.

### PR DESCRIPTION
**Pull request by Lukas1h**
`feeds/friends` was removed recently. It's now replaced with `feeds/friends-v1`. This newer feed supports multiple posts per moment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,77 +274,147 @@ Response (Un-authenticated)
...
-        "date": {
-          "_seconds": 1660002563,
-          "_nanoseconds": 350000000
-        }
+        "secondary": {
+          "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+          "width": 2000,
+          "height": 1500
+        },
+        "caption": "I love my tortilla blanky 🥰",
```
Comment (userbradley) [inline README.md:309]: I don't know the context, but I can get on board with this

File: README.md [hunk 2]
```diff
@@ -274,77 +274,147 @@ Response (Un-authenticated)
...
+            "height": 2000
+          },
+          "secondary": {
+            "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+            "width": 1500,
+            "height": 2000
+          },
+          "location": {
+            "latitude": 43.079,
+            "longitude": -123.5797
```
Comment (userbradley) [inline README.md:377]: I have checked and this is just on a highway, so not too worried about exposing someones location here

**Review by userbradley — APPROVED** [review]

**Comment by userbradley** [comment]
@Lukas1h I'm going to merge this, but do you mind emailing me when you get a sec? Interested to see how you found out about the new API as I have been struggling (along with others) to get access to the app since SSL pinning was enabled

File: README.md (hunk 1 above)
Comment (Lukas1h) [reply README.md:309]: Context: https://cdn.bereal.network/Photos/zRHCCjGLIZZ7EnliQwSMZ6zacj23/post/GkZr_Wx0jeCJ7uFu.webp

File: README.md (hunk 2 above)
Comment (Lukas1h) [reply README.md:377]: Thanks for that. Totally forgot about location data. 🤦‍♂️

**Pull request merged**